source/content.ts

```diff
@@ -65,6 +65,8 @@ import './features/navigate-pages-with-arrow-keys';
 import './features/global-discussion-list-filters';
 import './features/filter-comments-by-you';
 import './features/sort-issues-by-update-time'; // Must be after global-discussion-list-filters and filter-comments-by-you
+import './features/add-updated-date-to-pinned-issues';
```

Comment:
```suggestion
import './features/pinned-issues-update-time';
```

Also there's an extra space below